pkgs/applications/misc/taskwarrior/default.nix

```diff
@@ -1,34 +1,27 @@
-{ lib, stdenv, fetchurl, cmake, libuuid, gnutls, python3, bash }:
+{ lib, stdenv, fetchFromGitHub, cmake, libuuid, gnutls, python3, xdg-utils }:
 
 stdenv.mkDerivation rec {
   pname = "taskwarrior";
-  version = "2.5.3";
-
-  srcs = [
-    (fetchurl {
-      url = "https://github.com/GothenburgBitFactory/taskwarrior/releases/download/v${version}/${sourceRoot}.tar.gz";
-      sha256 = "0fwnxshhlha21hlgg5z1ad01w13zm1hlmncs274y5n8i15gdfhvj";
-    })
-    (fetchurl {
-      url = "https://github.com/GothenburgBitFactory/taskwarrior/releases/download/v${version}/tests-${version}.tar.gz";
-      sha256 = "165xmf9h6rb7l6l9nlyygj0mx9bi1zyd78z0lrl3nadhmgzggv0b";
-    })
-  ];
-
-  sourceRoot = "task-${version}";
+  version = "2.6.0";
+
+  src = fetchFromGitHub {
+    owner = "GothenburgBitFactory";
+    repo = "taskwarrior";
+    rev = "v${version}";
+    sha256 = "sha256-2wVjRecfIlNFAxXFaiKfxy9zArDIYDTfDdrnSM7H8fM=";
+    fetchSubmodules = true;
+  };
 
-  postUnpack = ''
-    mv test ${sourceRoot}
+  postPatch = ''
+    sed 's#xdg-open#${lib.getBin xdg-utils}/xdg-open#' \
+      --in-place src/commands/CmdNews.cpp
```

Comment:
Lets use substituteInPlace.

Reply:
Also it should be `${lib.getBin xdg-utils}/bin/xdg-open`. Now fixed.